src/csharp/Microsoft.Spark.E2ETest/IpcTests/Sql/SparkSessionTests.cs

```diff
@@ -65,5 +67,55 @@ public void TestSignaturesV2_4_X()
         {
             Assert.IsType<SparkSession>(SparkSession.Active());
         }
+
+        /// <summary>
+        /// Test CreateDataFrame API.
+        /// </summary>        
+        [Fact]
+        public void TestCreateDataFrame()
+        {                                   
+            var schema = new StructType(new List<StructField>()
+            {
+                new StructField("Name1", new StringType()),
+                new StructField("Name2", new StringType())
+            });
+
+            List<GenericRow> data = new List<GenericRow>();
+            data.Add(new GenericRow(new object[] { "Alice", "Harry" }));
+            data.Add(new GenericRow(new object[] { "Bob", "Mary" }));
+
+            // Calling CreateDataFrame with schema
+            DataFrame df1 = _spark.CreateDataFrame(data, schema);
+            Assert.IsType<DataFrame>(df1);
+            Assert.Equal<StructType>(schema, df1.Schema());
+            var df1Collect = df1.Collect();
```

Comment:
```suggestion
            Row[] rows = df1.Collect().ToArray();
```

Reply:
This throws an error - does not contain definition for `ToArray()`

Reply:
You'll need to add the LInq namespace.  https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.toarray?view=netcore-3.1

Reply:
Done, thanks!